cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -472,6 +475,10 @@ impl OpenshiftSecondaryMetadataParserPlugin {
 #[async_trait]
 impl InternalPlugin for OpenshiftSecondaryMetadataParserPlugin {
     async fn run_internal(self: &Self, mut io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("parse-secondary-metadata".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```